lerna.json

```diff
@@ -1,18 +1,29 @@
 {
-  "lerna": "^4.0.0",
   "npmClient": "yarn",
   "useWorkspaces": true,
   "packages": [
```

Comment:
For some reason if you don't specify each package then lerna can't figure it out. I _think_ it might be because we have packages with the same name in different locations.